package.json

```diff
@@ -26,6 +26,7 @@
   "scripts": {
     "build:clean": "rm -rf dist",
     "build": "pnpm build:clean && pnpm run _build:types && pnpm _build:esm && pnpm _build:cjs && pnpm _build:browser",
+    "build:tsup": "tsup",
```

Comment:
What's this one for?

Reply:
This would replace pnpm build if you like it